docs/Actions/revoke-user-role.md

```diff
@@ -0,0 +1,197 @@
+---
+id: revoke-user-role
+title: Revoke User Role
+sidebar_label: Revoke User Role
+---
+
+> Audience: [`Business Users`](/docs/audience#business-users)<br/>
+> Skill Prerequisites: `Permission Scheme`
+
+## `Typical Use Cases`
+
+- Revoke roles of existing users
+- Remove access to resources for specific users
+
+## `Don't use it to`
+
+- Prevent users from accessing the system at all. Instead, mark users as deleted or unauthorize them
+
+## `Related Actions`
+
+| Action Name                                             | Description                                                         |
+| ------------------------------------------------------- | ------------------------------------------------------------------- |
+| [Load Users](/docs/Actions/load-user.md)                | Can be used to revoke roles to a single user or to a list of users. |
+| [User Registration](/docs/Actions/user-registration.md) | Can be used to revoke roles to a newly created user.                |
+
+## `Input Parameter Reference`
+
+| Parameter        | Description                                                                              | Supports Variables |
+| ---------------- | ---------------------------------------------------------------------------------------- | ------------------ |
+| Role             | Select the security role, from existing roles, to revoke.                                | Yes                |
+| Other Role Names | Freely input the role name(s) you want to revoke. Multiple roles are separated by comma. | Yes                |
+
+## `Examples`
+
+### `1. Revoke a User Role `
```

Comment:
It's a dangerous example. If a Citizen Developer runs this sample, they're going to lock themselves out. Let's do the example with the Users role instead.

Reply:
My bad, I changed the example.